tests/v2/validation/networking/connectivity/network_policy.go

```diff
@@ -167,8 +167,8 @@ func isCloudManagerEnabled(client *rancher.Client, clusterID string) (bool, erro
 	return true, nil
 }
 
-// isNodePool is a helper function that checks if the machine pool cluster size is greater than or equal to 3
-func isNodePool(steveClient *steveV1.Client) (bool, error) {
+// IsNodePoolSizeValidSizeValid is a helper function that checks if the machine pool cluster size is greater than or equal to 3
```

Comment:
`IsNodePoolSizeValidSizeValid` SizeValid is added twice, can you update it to `IsNodePoolSizeValid`

Reply:
It is true, sorry about that